gen/definitions/profile_parcels/service_lan_vpn.yaml

```diff
@@ -0,0 +1,423 @@
+---
+name: Service LAN VPN
+rest_endpoint: /v1/feature-profile/sdwan/service/%v/lan/vpn
+minimum_version: 20.12.0
+test_tags: [SDWAN_2012]
+attributes:
+  - tf_name: feature_profile_id
+    reference: true
+    type: String
+    mandatory: true
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: sdwan_service_feature_profile.test.id
+  - model_name: vpnId
+    tf_name: vpn
+    example: 1
+  - model_name: name
+    tf_name: config_description
```

Comment:
Why do we call this `config_description`?

Reply:
In the UI this is under "Basic Configuration" and is called "description" but we can't call it description as it would conflict the parcel description